tst/EnergyPlus/unit/OutputProcessor.unit.cc

```diff
@@ -3355,6 +3355,72 @@ namespace EnergyPlus {
 			} ) );
 
 		}
+		TEST_F (SQLiteFixture, OutputProcessor_GenOutputVariablesAuditReport)
```

Comment:
It was intended to be part of this PR but you are right it does not test Meter:Custom. I decided to test another part of outputprocessor.cc that wasn't yet covered by any unit tests.